R/impute_knn.R

```diff
@@ -118,7 +118,7 @@ step_impute_knn <-
     if (length(options) > 0) {
       if (any(!(opt_nms %in% c("eps", "nthread")))) {
```

Comment:
`rlang::arg_match()`?

Reply:
I'm planning to add a helper function for this as we pass through additional arguments in a list a few places
https://github.com/tidymodels/recipes/issues/1269